### PR TITLE
Unit test: name, organization, success scenario

### PR DESCRIPTION
**Pull request by judubjazz**

### CODE REVIEW

File: api/V1/middlewares/user_roles/role-check-valid.js [hunk 1]
```diff
@@ -3,17 +3,12 @@ module.exports = (req, res, next) => {
...
     error.status = 422;
     next(error);
-  }
-
-  next();
-};
 
-module.exports = (req, res, next) => {
-  if (typeof req.body.organization_id === 'undefined') {
+  }else if (typeof req.body.organization_id === 'undefined') {
```
Comment (ethienneroy) [inline api/V1/middlewares/user_roles/role-check-valid.js]: Met un espace entre l’acolade et le else.

File: api/V1/middlewares/user_roles/role-check-valid.js [hunk 2]
```diff
@@ -3,17 +3,12 @@ module.exports = (req, res, next) => {
...
     const error = new Error('Missing organization_id');
     error.status = 422;
     next(error);
+  }else{
```
Comment (ethienneroy) [inline api/V1/middlewares/user_roles/role-check-valid.js]: Ici aussi

File: api/V1/middlewares/user_roles/role-check-valid.js (hunk 1 above)
Comment (ethienneroy) [inline api/V1/middlewares/user_roles/role-check-valid.js]: Met un espace entre l’accolade et le else

File: api/V1/middlewares/user_roles/role-check-valid.js (hunk 2 above)
Comment (ethienneroy) [inline api/V1/middlewares/user_roles/role-check-valid.js]: Ici aussi

**Comment by judubjazz** [comment]
Ouais j vais setter ca demain. Desole pour les console.log je vais m assurer de tous les deleter dorenavant

Sent from my iPhone

On Jun 13, 2018, at 10:54 PM, ethienneroy <notifications@github.com<mailto:notifications@github.com>> wrote:


@ethienneroy commented on this pull request.

Tu devrais utiliser un forrmqter pour ne plus avoir a te soucier de l’indentation de ton code

________________________________

In api/V1/middlewares/user_roles/role-check-valid.js<https://github.com/jbergeron03/node-login/pull/20#discussion_r195290276>:

>
-module.exports = (req, res, next) => {
-  if (typeof req.body.organization_id === 'undefined') {
+  }else if (typeof req.body.organization_id === 'undefined') {


Met un espace entre l’acolade et le else.

________________________________

In api/V1/middlewares/user_roles/role-check-valid.js<https://github.com/jbergeron03/node-login/pull/20#discussion_r195290291>:

>      const error = new Error('Missing organization_id');
     error.status = 422;
     next(error);
+  }else{


Ici aussi

________________________________

In api/V1/middlewares/user_roles/role-check-valid.js<https://github.com/jbergeron03/node-login/pull/20#discussion_r195290446>:

>
-module.exports = (req, res, next) => {
-  if (typeof req.body.organization_id === 'undefined') {
+  }else if (typeof req.body.organization_id === 'undefined') {


Met un espace entre l’accolade et le else

________________________________

In api/V1/middlewares/user_roles/role-check-valid.js<https://github.com/jbergeron03/node-login/pull/20#discussion_r195290460>:

>      const error = new Error('Missing organization_id');
     error.status = 422;
     next(error);
+  }else{


Ici aussi

**Pull request merged**